clojure-ts-mode.el

```diff
@@ -966,6 +966,9 @@ See `clojure-ts--font-lock-settings' for usage of MARKDOWN-AVAILABLE."
       (when (fboundp 'transpose-sexps-default-function)
         (setq-local transpose-sexps-function #'transpose-sexps-default-function)))))
 
+(when (fboundp #'derived-mode-add-parents)
```

Comment:
I'd suggest adding a comment here explaining this, given that this API is fairly new and unknown.

Reply:
Thank you, I've done.